common/djangoapps/static_replace/models.py

```diff
@@ -27,3 +27,27 @@ def __repr__(self):
 
     def __unicode__(self):
         return unicode(repr(self))
+
+
+class AssetExcludedExtensionsConfig(ConfigurationModel):
+    """Configuration for the the excluded file extensions when canonicalizing static asset paths."""
+
+    class Meta(object):
+        app_label = 'static_replace'
+
+    excluded_extensions = TextField(
+        default='html',
+        help_text='The file extensions to exclude from canonicalization.  No leading period required. ' +
+        'Values should be space separated i.e. "html svg css"'
+    )
+
+    @classmethod
```

Comment:
That makes sense, but also, this would be bucking the way all over ConfigurationModel-based configs actually work.  They all use the `@classmethod`/`cls.current()` approach.